packages/flame/test/game/mixins/keyboard_test.dart

```diff
@@ -3,27 +3,23 @@ import 'package:flame/input.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:mocktail/mocktail.dart';
 
 class _ValidGame extends FlameGame with KeyboardEvents {}
 
 class _InvalidGame extends FlameGame
     with HasKeyboardHandlerComponents, KeyboardEvents {}
 
-class _MockRawKeyEventData extends Mock implements RawKeyEventData {
-  @override
-  String toString({DiagnosticLevel minLevel = DiagnosticLevel.debug}) {
-    return super.toString();
-  }
-}
-
 void main() {
   group('Keyboard events', () {
     test(
       'game with KeyboardEvents can handle key events',
       () {
         final validGame = _ValidGame();
-        final event = RawKeyDownEvent(data: _MockRawKeyEventData());
+        const event = KeyDownEvent(
+          physicalKey: PhysicalKeyboardKey.arrowUp,
```

Comment:
I was unsure what to do here, I figured the up arrow is a generic enough key event for a game to process...